src/main/java/org/truffleruby/RubyContext.java

```diff
@@ -101,6 +105,7 @@
 
     private final CompilerOptions compilerOptions = Truffle.getRuntime().createCompilerOptions();
 
+    private final Random random;
```

Comment:
Can it be typed as `SecureRandom` then to clarify?

Reply:
I don't get it - users of the random don't need to know about how we produce the random values.

Reply:
It's important that the values are secure, so I don't see why we should hide it in the field information.
I think we should not expose the SecureRandom instance anyway (see below).